src/main/java/com/urbancode/jenkins/plugins/ucdeploy/VersionHelper.java

```diff
@@ -295,6 +295,8 @@ private void setComponentVersionProperties(
                 propSheetDef = compClient.getComponentVersionPropSheetDef(component);
                 propSheetDefId = (String) propSheetDef.get("id");
                 propSheetDefPath = (String) propSheetDef.get("path");
+                propSheetDefPath = propSheetDefPath.replaceAll("/", "&");
```

Comment:
In addition to the email I sent, could you run a simple `echo $PATH` command? There is a good chance there is some environmental setting that is conflicting with our encoding.